src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,296 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //================================================
+  //MAIN CONTAINER
+  //================================================
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(600px);
+    padding-top: $spacing-3xs; // allow space for focus styles
+
+    .#{$prefix}--search-input {
+      background-color: transparent;
+    }
+  }
+
+  //================================================
+  //TABLE HEADER
+  //================================================
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
+    color: $text-01;
+    @include font-size('20');
+    font-weight: 400;
+    padding: 22px 0 0 $spacing-md;
+    position: relative;
+    top: rem(21px);
+  }
+
+  .#{$prefix}--data-table-v2-optional {
+    position: relative;
+    left: rem(17px);
+    top: rem(-20px);
+    @include font-size('14');
+    color: $text-02;
+  }
+
+  //================================================
+  //DATA TABLE - DEFAULT STYLES
+  //================================================
+  .#{$prefix}--data-table-v2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+
+    [class*='col'] {
+      outline: none;
+    }
+
+    thead {
+      @include type-style('body-short-01');
+      background-color: $ui-03;
+      font-weight: 700;
```

Comment:
`font-weight` should be a part of the type style, I think, might need to figure out which token design wants to use